x-pack/plugins/apm/server/lib/alerts/register_error_count_alert_type.ts

```diff
@@ -136,7 +143,7 @@ export function registerErrorCountAlertType({
 
             return {
               serviceName: latest['service.name'] as string,
-              environment: latest['service.environment'] as string | undefined,
+              environment: latest['service.environment'] as string,
```

Comment:
this can still be `null`. But I don't think we need the top_metrics agg, we can use the keys from the multi_terms bucket. I think I used top_metrics because it was easier than dealing with how multi_terms returns its keys.

Reply:
How can it be `null` when `missing` is specified?

![image](https://user-images.githubusercontent.com/209966/122361138-43b56d80-cf57-11eb-9164-3beca537ca29.png)

Reply:
Ah, I see what you are saying about top_metrics now

Reply:
Removed the top_metrics agg